From f52c40680f0aad44b9ae16648803453ec00cbb2c Mon Sep 17 00:00:00 2001
From: Paul Crawford <psc@sat.dundee.ac.uk>
Date: Fri, 30 Dec 2016 15:55:45 +0000
Subject: [PATCH 1/3] Compile with musl when nfs is disabled

musl does by default not ship with rpc headers. The watchdog should
not require rpc headers when nfs support is disabled.

Patch by Felix Janda <fjanda@users.sf.net>
---
 include/sundries.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/sundries.h b/include/sundries.h
index 4379982..98c489a 100644
--- a/include/sundries.h
+++ b/include/sundries.h
@@ -9,9 +9,11 @@
 #include <signal.h>
 #include <stdarg.h>
 #include <stdlib.h>
+#if HAVE_NFS
 #if !defined(bool_t) && !defined(__GLIBC__)
 #include <rpc/types.h>
 #endif
+#endif
 
 extern int mount_mount_quiet;
 extern int mount_verbose;
-- 
2.7.3

